pkg/components/contour/template.go

```diff
@@ -45,5 +45,276 @@ spec:
   selector:
     app: envoy
   type: LoadBalancer
+`,
+
+	"03-contour.yaml": `
```

Comment:
Why not create small helm chart and use chart templating?

Reply:
So I tried moving the contour component helm chart. You can find the work done for it here:
https://github.com/kinvolk/lokomotive/pull/new/surajssd/move-contour-to-helm-chart

The problem with it is that we ship grafana dashboard and alertmanager config which conflicts with the helm's templating when used directly.

```console
$ helm template .
Error: parse error at (contour/templates/grafana-dashboard.yaml:103): function "namespace" not defined
```

So these dashboards should either be moved to separate json files and then refer them from the config files. Or escape the conflicting `{{}}` brackets and keep them where the config is. Some inspiration is needed from other charts.